### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by FlyingUnicornSF**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (jayrbolton) [inline .vscode/settings.json]: Remove and gitignore this file plz

File: src/client/index.tsx [hunk 2]
```diff
@@ -17,8 +17,11 @@ const CONTAINER = document.getElementById('react-root');
 
 // Underline the current item in the top-nav. Plain JS. Uses tachyons classes.
 document.querySelectorAll('[data-hl-nav]').forEach(node => {
+  let HTMLEle:HTMLElement = node as HTMLElement;
+  HTMLEle.style
```
Comment (jayrbolton) [inline src/client/index.tsx]: What is this line?

File: src/client/js/components/dashboard/NarrativeList/ItemList.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
+import React, {Component} from 'react';
```
Comment (jayrbolton) [inline src/client/js/components/dashboard/NarrativeList/ItemList.js]: Remove this file

File: src/client/index.tsx (hunk 2 above)
Comment (FlyingUnicornSF) [reply src/client/index.tsx]: querySelectorAll returns an Element and NOT HTMLELement. If you want to use .style, then you have to convert  it to HTMLElement

Comment (jayrbolton) [reply src/client/index.tsx]: But the line is just `HTMLEle.style` -- it doesn't do anything

**Pull request merged**